src/overrides/safari_version.py

```diff
@@ -0,0 +1,44 @@
+import os
+import subprocess
+import plistlib
+
+
+_safari_version = ""
+_SAFARI_BUNDLE = "com.apple.Safari"
+
+
+def get() -> str:
+    global _safari_version
+    if _safari_version != "":
+        return _safari_version
+
+    try:
+        _safari_version = _get_app_version_by_bundle_id(_SAFARI_BUNDLE)
+    except Exception as e:
+        _safari_version = "unknown"
+        print("Exception retrieving safari version:", e.__class__, e)
```

Comment:
If you remove the global you can just return here and you don't need the finally block.